submission_rules.adoc

```diff
@@ -535,8 +535,26 @@ Each submitter is required to review at least one other submission. Required rev
 
 ### Auditing
 
-TBD
+#### Training
+
+In order to reduce the burden on the submitter as well as the Submitter’s Working Group (SWG) during the review period, submitters shall ensure compliance with RCP tests ahead of the submission deadline. Submissions that need new RCPs are required to supply those RCPs at the same time as their submission, as specified by the Logging Library’s RCP support. While providing new RCPs, a submitter must also include reference run logs for the SWG and reference owner to review.
```

Comment:
Looks great. I am approving, but since RCPs are documented in the training rules document, if you could point the reader to that it would be better. The wording now points the submitters to the logging library which does not contain comprehensive with examples.

Reply:
done